How-to-find-the-commit-which-introduced-a-bug.md

```diff
@@ -0,0 +1,181 @@
+## Finding Bad Commits with the help of `git blame` and `git bisect`
+
+#### Before using both these commands:
+1. **Make sure to checkout the latest commit and see if the error exists.**
+    * **If the error is not reproducible on the latest commit:** we don’t need to do anything! You can post video proof on the issue thread and request that a maintainer close the issue.
+    * **Otherwise**: continue with the steps below.
+2. **Depending on how far back you want to go, checkout one of the following:**
+    * **This commit from Dec 2022 &lt;[9a334e9bde1d3d10e3b69dcd461d3e649733b0c0](https://github.com/oppia/oppia/commit/9a334e9bde1d3d10e3b69dcd461d3e649733b0c0)>, which is when the Python version was last updated – though this might not work if you are using Docker.**
```

Comment:
Why do you still have html constructs (&lt;) here and below? 

Perhaps just put them in parentheses: (...)

Reply:
Removed the constructs and added paranthesis